exporter/containerimage/writer.go

```diff
@@ -92,8 +101,26 @@ func (ic *ImageWriter) Commit(ctx context.Context, inp *exporter.Source, session
 		}
 	}
 
-	if len(inp.Refs) == 0 {
-		remotes, err := ic.exportLayers(ctx, opts.RefCfg, session.NewGroup(sessionID), inp.Ref)
+	if !multiPlatform {
+		if len(p.Platforms) > 1 {
+			return nil, errors.Errorf("unable to export without multi-platform, multiple platforms passed to exporter")
```

Comment:
Not sure about the wording, maybe smth like "cannot export multiple platforms without multi-platform enabled"?